doc/logging.md

```diff
@@ -0,0 +1,19 @@
+# How does Happy test log work
+
+## Types of Happy logs
+
+  * Debug logs - Logs related to Happy's role as test coordinator and orchestrator, where it facilitates the activities of virtual device nodes.
+
+  * Device logs - Logs related to software such as openweave running on individual virtual device nodes within a Happy test network. OpenWeave is one of the software example, device logs will have OpenWeave logs.
+
+## Where Happy logs are stored
+
+  * If `HAPPY_LOG_DIR` is found in os environment variable, Happy uses `HAPPY_LOG_DIR` value as log folder location to store Happy test logs.
```

Comment:
If the `HAPPY_LOG_DIR` environment variable exists, Happy uses the `HAPPY_LOG_DIR` value...